lib/API/Log.js

```diff
@@ -98,9 +98,11 @@ Log.stream = function(Client, id, raw, timestamp, exclusive, highlight) {
     var min_padding = 3
 
     bus.on('log:*', function(type, packet) {
-      if (id !== 'all'
-          && packet.process.name != id
-          && packet.process.pm_id != id)
+      var isMatchingProcess = id === 'all'
+        || packet.process.name === id
+        || packet.process.pm_id === id
+        || packet.process.namespace === id;
+      if (!isMatchingProcess)
```

Comment:
separating out `isMatchingProcess` felt more readable to me - happy to revert back to something like this instead:

```js
        if (id !== 'all'
          && packet.process.name != id
          && packet.process.pm_id != id
          && packet.process.namespace != id)
```